### DISCUSSION TITLE
Cryptocurrency addresses

### DISCUSSION THREAD

**Discussion by trymeouteh** [JulianPrieber/littlelink-custom#677] [Feature requests]
A new link type which users can share a crypto address.

When the user clicks on the link, a mod will appear with the address text displayed with a copy button next to it. And a QR of the address to be displayed.

**Comment by JulianPrieber** [comment]
I think we have a copy text block on our planned features list, would that work?

I think someone also suggested a crypto block with a scannable QR code.
